modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -32,13 +36,38 @@ have to contact Red Hat support with your installation logs.
 section of this topic and save it as `04_bootstrap.py` on your computer. This
 template describes the bootstrap machine that your cluster requires.
 
-. Export the following variables required by the resource definition:
+. Export the variables that the deployment template uses:
+.. Export the control plane subnet location:
 +
+ifndef::shared-vpc[]
 ----
 $ export CONTROL_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-master-subnet --region=${REGION} --format json | jq -r .selfLink`
+----
+endif::shared-vpc[]
+ifdef::shared-vpc[]
+----
+$ export CONTROL_SUBNET=`gcloud compute networks subnets describe <master-subnet> --region=${REGION} --format json | jq -r .selfLink`
+----
+Where `<master-subnet>` is the name of the subnet in the shared VPC to deploy the control plane machines to.
+endif::shared-vpc[]
+
+.. Export the location of the {op-system-first} image that the installation program requires:
++
+----
 $ export CLUSTER_IMAGE=`gcloud compute images describe ${INFRA_ID}-rhcos-image --format json | jq -r .selfLink`
+----
+
+.. Export each zone that the cluster uses:
++
+----
 $ export ZONE_0=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[0] | cut -d "/" -f9`
+----
++
+----
 $ export ZONE_1=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[1] | cut -d "/" -f9`
+----
++
+----
```

Comment:
The bootstrap deployment template is changed. please load the latest from https://github.com/openshift/installer/blob/master/upi/gcp/04_bootstrap.py